lesson06/assignment/scripts/script.js

```diff
@@ -0,0 +1,13 @@
+/*user hovers on nav item and shooting stars fly across screen, as well as yellow orb appearing around nav item*/
+
+$('.nav__link').hover(function(e) {
+  $('#starshooter1').addClass('main__starshooter1');
+  $('#starshooter2').addClass('main__starshooter2');
+  $('#starshooter3').addClass('main__starshooter3');
+  $(this).addClass('nav__link--hover');
+}, function() {
+  $('#starshooter1').removeClass('main__starshooter1');
+  $('#starshooter2').removeClass('main__starshooter2');
+  $('#starshooter3').removeClass('main__starshooter3');
+  $(this).removeClass('nav__link--hover');
+});
```

Comment:
The shooting star animation is so delightful. Just an excellent user experience.

The style change on hover of nav elements can be accomplished with the CSS pseudoselector `:hover`. This is preferable to adding and removing a class using javascript, because it's simpler code and keeps style logic contained in the stylesheets.

Since only index.html has the startshooter elements to use this animation, if the hover styles were moved to SCSS file, the other pages in the site wouldn't need to import this file or jquery, resulting in faster load times and less data sent. Less data = less cost to users on mobile networks that have data usage caps.